deploy_tools/cli.py

```diff
@@ -74,20 +81,22 @@ def validate_address(ctx, param, value):
     help="automatically determine the nonce of first transaction to be sent",
     default=False,
     is_flag=True,
+    envvar="DEPLOY_TOOLS_AUTO_NONCE",
```

Comment:
Wouldnt that mean that if I use this for a different command line programm, like `auction-deploy`, that I still have to use `DEPLOY_TOOLS...`. Is that what we want? Or should we provide a way to easy change the prefix?
What do you think @schmir-at-bb?

Reply:
I am fine with the current behaviour. Maybe we can find a better prefix than DEPLOY_TOOLS_ in order to 'decouple' it from the deploy-tools package a bit.

Reply:
Yeah, I though about this as well but ended up with not worth the work. But I totally understand the point. Maybe @schmir-at-bb his approach with more decoupled names is already enough. I'm open for suggestions.
I mean the crass approach would be to remove the prefix at all on go for the argument names only. It maybe depends on the use case you had in mind. If you want to have such variables at the top level within your `.zshenv` for (an extreme) example I wouldn't like to generalize them too much. Having them for more convenience on end-to-end test like purpose such simple names work for me.

Reply:
Decided for deleting the prefix. See comments on this PR below.